### PR TITLE
add light-java, light-java-rest and light-oauth2

### PR DESCRIPTION
**Pull request by stevehu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -549,6 +549,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Eureka](https://github.com/Netflix/eureka) - REST based service registry for resilient load balancing and failover.
 * [Lagom](https://www.lightbend.com/lagom) - Framework for creating microservice-based systems.
 * [SnopEE](https://github.com/ivargrimstad/snoop) - Discovery service for Java EE microservices.
+* [Light-Java](https://github.com/networknt/light-java) - The fastest and most productive microservices framework
```
Comment (akullpp) [inline README.md]: The fastest and most productive is extremely subjective...

**Comment by akullpp** [comment]
Is there an overview of the light framework? If so, please add it to the Plattform section which is meant for frameworks that offer multiple libraries otherwise the list will become too polluted.

**Comment by stevehu** [comment]
@akullpp I have remove the entry for web framework as we are focusing on microservices architecture and updated the descriptions for three products. Thanks.

**Comment by akullpp** [comment]
@stevehu Can you please add the generic website/page for the parent project to the Plattform section instead of splitting it up?

**Comment by stevehu** [comment]
@akullpp Now I understand what do you mean. Please review and let me know if you have questions. Thanks for your help.

File: README.md [hunk 2]
```diff
@@ -656,6 +655,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 * [CUBA Platform](https://cuba-platform.com) - High level framework for development of enterprise applications with rich web interface, based on Spring, EclipseLink and Vaadin.
 * [Spring](http://projects.spring.io/spring-framework/) - Provides many packages ranging from dependency injection to aspect-oriented programming to security.
+* [Light-Java](https://github.com/networknt/light-java) - A fast, light weight and productive microservices framework with [security](https://github.com/networknt/light-oauth2) built in.
```
Comment (jbduncan) [inline README.md]: Hi @stevehu, minor nitpick - I think it should say "lightweight" rather than "light weight". :)

**Comment by stevehu** [comment]
@jbduncan Thanks for the quick review and response. It was a typo.

**Pull request merged**